orchestrator/lib/coolfhir/azure.go

```diff
@@ -54,3 +55,48 @@ func (a azureTokenSource) Token() (*oauth2.Token, error) {
 		Expiry:      accessToken.ExpiresOn,
 	}, nil
 }
+
+type FHIRRoundTripperConfig struct {
```

Comment:
Naming is hard, please let me know if there is something better I can name these config structs and the code below

Reply:
as the package is already "fhir", I'd drop the "FHIR" prefix. Other suggestion: "ClientConfig"?